File: C/Exercicio24.c
#include <stdio.h>
#include <stdlib.h> 

int main(int argc, char *argv[]){
  
  char sexo;
  
  printf("Digite o sexo(F/M): ");
  scanf("%c", &sexo);

  while (sexo != 'F' && sexo != 'M'){
    printf("Erro! Selecionar uma das opcoes de sexo disponiveis (F/M): ");
    scanf("%c", &sexo); 
  }
    
  return 0;
}

File: Python/Exercicio24.py
sexo = (input("Digite o sexo (F/M): "))

while sexo != 'F' and sexo != 'M':
    sexo = (input("Erro! Digite M ou F: "))

if sexo == 'F' or sexo == 'M':
    print("Sexo", sexo, "digitado!")